pages/Profile/Change-phone-number.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <div>
+    <!-- enter phone number -->
+    <div
+      v-if="status === 'enterPhoneNumber'"
+      class="w-[530px] h-[422px] rounded-lg overflow-hidden text-sm"
+    >
+      <div
+        class="w-full h-[51px] bg-base-200 flex items-center justify-start text-lg textco"
+      >
+        <div class="w-[2px] h-5 rounded-sm bg-icon ml-4"></div>
+        <p>تغییر شماره همراه</p>
+      </div>
+      <div
+        v-if="status === 'enterPhoneNumber'"
+        class="w-full h-[371px] bg-white flex flex-col justify-between items-center p-3"
+      >
+        <p>
+          کاربر گرامی برای تایید شماره همراه خود لطفا شماره همراه خود را در کادر
+          پایین وارد کنید تا ما برای شما یک کد تایید ارسال کنیم :<br />
+          شماره همراه شما برای امنیت حساب کاربریتون مهم است
+        </p>
+        <div class="w-full h-auto px-10">
+          <p class="text-base-500 text-xs">شماره همراه</p>
+          <input
+            class="w-full h-11 text-left border rounded border-base-300 p-2 focus:border-b-dimound focus:border-b-2"
+            placeholder="09"
+            type="tel"
+            v-model="phoneNumber"
+          />
+        </div>
+        <button
+          @click="changeStep"
+          class="w-[137px] h-10 rounded-lg bg-dimound text-white"
+        >
+          دریافت کد تایید
+        </button>
+      </div>
+    </div>
+    <!-- confirm code -->
+    <div
+      v-if="status === 'enterVerificationCode'"
+      class="w-[530px] h-[422px] rounded-lg overflow-hidden text-sm"
+    >
+      <div
+        class="w-full h-[51px] bg-base-200 flex items-center justify-start text-lg textco"
+      >
+        <div class="w-[2px] h-5 rounded-sm bg-icon ml-4"></div>
+        <p>تغییر شماره همراه</p>
+      </div>
+
+      <div
+        class="w-full h-[371px] bg-white flex flex-col justify-between items-center p-3"
+      >
+        <p>
+          کد فعالسازی که برای شما ارسال شده است رو در کادر پایین بنویسید .
+          <br />اگر کدی برای شما ارسال نشده است دریافت مجدد کد را فشار دهید
+        </p>
+        <div
```

Comment:
There is a point that you ignore, after pressing any char, we have move the pointer to next input.
In addition to that, you have set the direction from left to right. Because when user pressed the `tab` key, we should move pointer to next input and it is on the right side. so do these:
- move pointer after pressing any key with a char to next input.
- move pointer to next input on the right side with `tab` key.